statsmodels/tsa/ardl/tests/test_ardl.py

```diff
@@ -821,3 +821,12 @@ def test_resids_ardl_uecm():
     uecm_res = uecm_mod.fit()
 
     assert_allclose(uecm_res.resid, ardl_res.resid)
+
+
+@pytest.mark.parameterize("y_lags",[None, 1, 2])
```

Comment:
Sorry, I had a typo in my code.  Should be `pytest.mark.parametrize`.  I'm going to merge and will fix myself.